### PR TITLE
link_updated in changelog

### LINKED ISSUE #5302: The Table of Contents in the changelog links to Github
On https://thanos.io/tip/thanos/changelog.md/ -> Table of Contents

This link [v0.25.2](https://github.com/thanos-io/thanos/tree/release-0.25) - 2022.03.24 should actually go towards https://thanos.io/tip/thanos/changelog.md/#v0252httpsgithubcomthanos-iothanostreerelease-025---20220324

Perhaps this page / table of contents could be easier/nicer as well - For example that the table of contents would be a menu on the left or right so you do not have to scroll all the way down. 

Furthermore, it would be nice if we link to github, but not towards https://github.com/thanos-io/thanos/tree/release-0.25 but to https://github.com/thanos-io/thanos/releases/tag/v0.25.2

### PR DESCRIPTION
**Pull request by Akshit42-hue**
Signed-off-by: Axit Patel <patelakshit2025@gmail.com>

Fixes #5302 
Update the link in changelog

### CODE REVIEW

**Comment by Akshit42-hue** [comment]
please do have a look @wiardvanrij

**Comment by Akshit42-hue** [comment]
Hey @wiardvanrij, can't able to find any leads in hugo.yaml file can you please provide some more leads

**Comment by bwplotka** [comment]
Hm... I think there is a simpler thing we can do to make it all work.

We can stop doing links, just keep a plain header. This will make the Table of Content work like a charm. (: But we have to be consistent and do that to all headers here, not only one

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Removed
 
-## [v0.25.2](https://github.com/thanos-io/thanos/tree/release-0.25) - 2022.03.24
+## [v0.25.2](https://thanos.io/tip/thanos/changelog.md/#v0252httpsgithubcomthanos-iothanostreerelease-025---20220324) - 2022.03.24
```
Comment (bwplotka) [inline CHANGELOG.md:46]:
```suggestion
## v0.25.2 - 2022.03.24

Release [branch](https://github.com/thanos-io/thanos/tree/release-0.25)

```

**Review by bwplotka — COMMENTED** [review]
I think this might work instead (proposed below), but we have to be consistent and do it everywhere.

File: CHANGELOG.md (hunk 1 above)
Comment (Akshit42-hue) [reply CHANGELOG.md:46]: Great suggestion @bwplotka

Comment (Akshit42-hue) [reply CHANGELOG.md:46]: @wiardvanrij What's your take? 
Should I go ahead with @bwplotka suggestion

**Comment by wiardvanrij** [comment]
PTAL at https://github.com/thanos-io/thanos/pull/5318#discussion_r862683163 comment. 

So, we change it in a way that the header does not contain the URL anymore. Which is good, so our website get's a nice table of contents. However, we still need to link to the release it self. 

```
## v0.25.2 - 2022.03.24
Release [branch](https://github.com/thanos-io/thanos/tree/release-0.25)
```

**Pull request closed without merging**